openlibrary/macros/AffiliateLinks.html

```diff
@@ -1,37 +1,39 @@
-$def with (opts)
+$def with (page, opts)
 
 $ prices = opts.get('prices')
 $ isbn = opts.get('isbn')
 $ asin = opts.get('asin')
 $ scribd = opts.get('scribd')
 
-$if scribd or isbn or asin:
-  <table class="buy-options-table">
-    <tbody>
-      $if asin:
-        $ amazon = "http://www.amazon.com/dp/%s/?tag=%s" % (asin, affiliate_id('amazon'))
-        <tr>
-          <td>
-            <a href="$amazon" title="Look for this edition for sale at Amazon" target="_blank">Amazon</a>
-          </td>
-          <td class="price">
-            $if prices and not is_bot():
-                $ amazon_metadata = get_amazon_metadata(isbn)
-                $if amazon_metadata and 'price' in amazon_metadata and amazon_metadata['price']:
-                    <span name="price">$(amazon_metadata['price'])</span>
-          </td>
-        </tr>
-      $if isbn and not is_bot():
-            <tr>
-              $ better_world_books = "http://www.anrdoezrs.net/links/%s/type/dlg/http://www.betterworldbooks.com/-id-%s.aspx" % (affiliate_id('betterworldbooks'), isbn)
-              <td><a href="$better_world_books" title="Look for this edition for sale at Better World Books" target="_blank">Better World Books</a></td>
-              <td class="price">
-              $if prices:
-                  $ betterworldbooks_metadata = get_betterworldbooks_metadata(isbn)
-                  $if betterworldbooks_metadata and 'price' in betterworldbooks_metadata and betterworldbooks_metadata['price']:
-                      $betterworldbooks_metadata['price']
-              </td>
-            </tr>
-      $if scribd and not (isbn or asin):
-        <tr><td><a href="http://www.scribd.com/doc/$scribd" title="Look for this edition for sale at Scribd" target="_blank">Scribd</a><br /><span class="gray smaller">eBook in browser</span></td></tr>
-  </table>
+<ul class="buy-options-table">
+  $if not is_bot():
+      <li>
+        $ bwb_affiliate_url = 'http://www.anrdoezrs.net/links/%s/type/dlg/http://www.betterworldbooks.com/' % affiliate_id('betterworldbooks')
```

Comment:
This link looks like it now functions with https. Last time I remember checking this anrdoezers.net linker, it didn't. There are possibly a few places this will need to be updated if https is now appropriate for this URL.

Reply:
Done (here)